package.json

```diff
@@ -2,15 +2,27 @@
   "name": "@waku/react",
   "version": "0.0.1",
   "description": "React hooks and components to use js-waku",
-  "types": "./dist/index.d.ts",
-  "module": "./dist/index.js",
+  "type": "module",
+  "main": "dist/index.cjs.js",
+  "module": "dist/index.esm.mjs",
+  "umd:main": "dist/index.umd.js",
+  "unpkg": "dist/index.umd.js",
```

Comment:
should we have an `unpkg` field in js-waku too? How does this help?

Reply:
Essentially it allows better integration with https://unpkg.com/
We might add it but that't not a big deal.